src/validation.cpp

```diff
@@ -4517,7 +4526,8 @@ bool CChainState::RewindBlockIndex(const CChainParams& params)
     // Loop until the tip is below nHeight, or we reach a pruned block.
     while (!ShutdownRequested()) {
         {
-            LOCK2(cs_main, ::mempool.cs);
+            LOCK(cs_main);
```

Comment:
nit:
```suggestion
            LOCK(::cs_main);
```

Reply:
Thanks. Will add the `::` when I have to retouch or on the next commit that touches those lines.